docs/server/commands.rst

```diff
@@ -901,6 +891,16 @@ run_cherrypy
 Run the CherryPy server bundled with the Translate Toolkit.
 
 
+.. django-admin:: start
+
+start
+^^^^^
+
+.. deprecated:: 2.7.3
```

Comment:
I still think that if it is gone, it is gone. So don't keep any unnecessary documentation for it.